cmd/cp-url.go

```diff
@@ -191,6 +191,9 @@ func makeCopyContentTypeC(sourceAlias string, sourceURL clientURL, sourceContent
 	newSourceSuffix := filepath.ToSlash(newSourceURL.Path)
 	if pathSeparatorIndex > 1 {
 		sourcePrefix := filepath.ToSlash(sourceURL.Path[:pathSeparatorIndex])
+		if sourceAlias == "" && targetAlias != "" {
```

Comment:
We should add a comment about this behavior difference @poornas

Reply:
done